api/database/migrations/2023_02_03_170212_add_empty_column_requests_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('pool_candidate_search_requests', function (Blueprint $table) {
+            $table->boolean('was_empty')->default(false)->after('pool_candidate_filter_id');
```

Comment:
just as a heads up, I don't think `after(...)` does anything for postgres
column ordering as a whole isn't pleasant in it [see this](https://stackoverflow.com/questions/285733/how-do-i-alter-the-position-of-a-column-in-a-postgresql-database-table)

![image](https://user-images.githubusercontent.com/40485260/217383340-a18e327e-86dc-4de6-8cf0-ed87636fd2d5.png)

Reply:
oh you are right, that is **_super_** annoying lol

Reply:
there is no table organization allowed here